core/pkg/evaluator/json_test.go

```diff
@@ -476,62 +476,6 @@ func TestResolveAllValues(t *testing.T) {
 	}
 }
 
-func TestMetadataResolveType(t *testing.T) {
```

Comment:
I mentioned the reason for deleting this in the description. This is covered [here](https://github.com/open-feature/flagd/pull/1702/files#diff-0cdd4fe716f4b8a94466279fd1b11187fcf4d74e6434727c33d57ed78c89fe27R452) now.